src/components/Signin.jsx

```diff
@@ -0,0 +1,215 @@
+                                        type="submit"
+                                        className="btn btn-outline-success rounded-pill shadow"
+                                        style={{width: "30%"}}
+                                    >
+                                        Send
+                                    </button>
+                                )}
+                                {status === "ERROR" && (
+                                    <h5 className="text-danger">Oops! There was an error.</h5>
+                                )}
+                            </div> */}
+                            <div className="form-group text-center">
+                              <Authenticate
+                                type="submit"
+                                className="btn btn-outline-success rounded-pill shadow"
+                              >
+                                SignIn
+                              </Authenticate>
+                              <ForgotPasswordButton
+                                type="submit"
+                                className="btn btn-outline-danger rounded-pill shadow"
+                              >
+                                Forgot your password?
+                              </ForgotPasswordButton>
+                            </div>
+                            <div className="form-group text-center">
+                              <label className="h5 contact-label">New to Social-Scheduler ? ,then Register here</label>
+                              <Authenticate
+                                type="submit"
+                                className="btn btn-outline-success rounded-pill shadow"
+                              >
+                                SignUp
+                              </Authenticate>
+                            </div>
+                            {/* social media links for login */}
+                            <div>
+                              <OR >
+                                <h2><span>OR</span></h2>
+                              </OR>
+                              <div className="col">
+                                <SocialMediaLinks >
+                                  <li>
+                                    <a href="#" className="fb btn">
+                                      <i className="fa fa-facebook fa-fw"></i> Login with Facebook
+                                    </a>
+                                  </li>
+                                  <li className="github-li">
+                                    <a href="#" className="github btn">
+                                      <i className="fa fa-github "></i> Login with Github
+                                    </a>
+                                  </li>
+                                  <li className="linkedin-li">
+                                    <LinkedinLink href="#" className="linkedin btn">
```

Comment:
Kindly make this a generic styled component for all social media icons.
Li
  - a
    - i
    - Text 
You will pass all the content to the component as prop.

Reply:
ok i will try

Reply:
bro how can u pass a title to a styled component , you can only pass the color.